code/go/0chain.net/mocks/state-context-i.go

```diff
@@ -0,0 +1,334 @@
+// Code generated by mockery v2.8.0. DO NOT EDIT.
```

Comment:
Why is this file here?

Reply:
This is not my change tbh, someone added it and not .gitignored it, will add to gitignore